addons/tasks/CfgTaskTypes.hpp

```diff
@@ -126,7 +126,7 @@ class CfgTaskTypes {
         displayName = CSTRING(Run);
     };
     class scout {
-        displayName = "$STR_A3_Role_Recon";
+        displayName = "$STR_A3_b_ctrg_soldier_tna_f0";
```

Comment:
This entry doesn't make sense for other languages.

Reply:
Probably need to add a stringtable key then. This was the only case correct one I found.